src/logic/operate.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/prefer-default-export */
+import Big from 'big.js';
+
+export const operate = (numberOne, numberTwo, operation) => {
+  let data = 0;
+  const num1 = Big(numberOne || '0');
+  const num2 = Big(numberTwo || '0');
+
+  switch (operation) {
+    case '÷':
+      try {
+        data = num1.div(num2);
+      } catch (e) {
+        data = 'ERROR';
+      }
+      break;
+    case 'X':
+      data = num1.times(num2);
+      break;
+    case '+':
+      data = num1.plus(num2);
+      break;
+    case '-':
+      data = num1.minus(num2);
+      break;
+    case '%':
+      data = num1.times(Big(0.01));
```

Comment:
When it comes to `%` we only need one operand.
example: click `10` then `%` = 0.01
This should work.
Example: `num1 = (num1 *= 0.01).toString();`

You might want to add `toString()` at the end of your cases.